core/heap.c

```diff
@@ -110,19 +110,26 @@ static const uint BLOCK_SIZES[] = {
     24, /* fcache empties and vm_area_t are now 20, vm area extras still 24 */
     /* 40 dbg / 36 rel: */
     ALIGN_FORWARD(sizeof(fragment_t) + sizeof(indirect_linkstub_t), HEAP_ALIGNMENT),
-#if defined(X64) || defined(CUSTOM_EXIT_STUBS)
-    sizeof(instr_t), /* 64 (96 x64) */
+#if defined(X64)
+    sizeof(instr_t), /* 104 x64 */
```

Comment:
It really went up 8 bytes?  Could your new field be placed somewhere else that only raises it 4?

Reply:
Couldn't find a place, just tried. By the way, this is slightly fragile, since a different compiler may pad differently, unless we are going to use attribute 'packed'.